core/src/main/scala/chisel3/package.scala

```diff
@@ -385,4 +385,7 @@ package object chisel3 {
   // Connection exceptions.
   case class BiConnectException(message: String) extends ChiselException(message)
   case class MonoConnectException(message: String) extends ChiselException(message)
+
+  final val deprecatedMFCMessage =
+    "this feature will not be supported as part of the migration to the MLIR-based FIRRTL Compiler (MFC). For more information about this migration, please see the Chisel ROADMAP.md."
```

Comment:
```suggestion
    "this feature will not be supported as part of the migration to the MLIR-based FIRRTL Compiler (MFC). For more information about this migration, please see the Chisel ROADMAP.md."
```

nit: why aren't we capitalizing `this`

Reply:
Well, I can actually see this going either way. The default sentence isn't capitalized in the error message. However, this is two complete sentences. I will update it. Thanks!

```
[warn] /Users/schuylere/repos/github.com/chipsalliance/chisel3/src/test/scala/chiselTests/AnnotatingDiamondSpec.scala:34:5: method apply in object annotate is deprecated (since Chisel 3.6): this feature will not be supported as part of the migration to the MLIR-based FIRRTL Compiler (MFC). For more information about this migration, please see the Chisel ROADMAP.md.
[warn]     annotate(anno)
[warn]     ^
```

LLVM/MLIR warnings often include a partial sentence in the message, e.g., `foo->emitError() << "is badf"` gets emitted as `firrtl.circuit op 'HerpDerp' is bad`. I think I got used to this and wrote the same thing here. 😕